### PR TITLE
Update UG with archive, unarchive and touch up list with new features…

### PR DESCRIPTION
**Pull request by alalal47**
… that were added.

Fix grammar issues and add newlines to improve readability.

### CODE REVIEW

**Comment by alalal47** [comment]
NOTE: any document that references help must be updated before the end

**Review by Cryolian — APPROVED** [review]
Agreed with the changes

File: docs/UserGuide.md [hunk 1]
```diff
@@ -272,12 +289,128 @@ undo
...
+Analysing Inputs...
+Directory loaded successfully!
+Data update success!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Runtime List: 
+Analysing Inputs...
+Directory loaded successfully!
+Data update success!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
```
Comment (tayponghee) [inline docs/UserGuide.md:405]: Looks great to me!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -304,6 +437,42 @@ sort payment
...
+Exits the Classify program.
+
+Format: `bye`
+```
+Example:
+bye
+Hope you've had a productive day. See you again! Goodbye!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+```
+
```
Comment (tayponghee) [inline docs/UserGuide.md:475]: Great addition, will need to modify the list function to include the 3 other types

**Review by tayponghee — APPROVED** [review]

**Pull request merged**